### ISSUE TITLE
Kill the application if we cannot connect to mongo

### ISSUE THREAD

**Issue by carmenfan** [3drepo/3drepo.io#4894]
### Description
(Pretty sure we used to do this!)

If we can no longer maintain a connection to the database, we should exit the application with an error so we can trigger the alert emails.

At the moment it returns an error on any db request and hopes to re-establish the connection the next time it has work. Which result in escaping from monitoring.

### Steps to replicate
- start the application
- make mongo inaccessible (e.g. turn off the service)

### Current Behaviour
- the application will continue to attempt to function

### Expected Behaviour
- it should exit so we get an alert and a new pod can spin up in replacement.

**Comment by carmenfan** [comment]
Verified that we do kill the application if we can't connect to mongo initially, and there is a auto recovery logic in place so if we struggled on a network blip, it will attempt to reconnect and function as normal

We have seen in prod logs that it has happened and manages to recover itself. So this i sworking as intended.

If we force crash on a disconnect, it may be worse as a short network blip will bring down all io pods. So we should really try to figure out what's the underlying problem here first.

Removing myself from this issue and waiting on further info.
